src/stories/Notice.stories.tsx

```diff
@@ -12,21 +13,29 @@ const NoticeStories: Meta = {
 
 export default NoticeStories;
 
-const Template: Story<NoticeProps> = ({
-  intent,
-  children,
-  onClose,
-}: NoticeProps) => {
+const Template: Story<NoticeProps> = ({ intent, children }: NoticeProps) => {
+  const [open, setOpen] = useState(true);
   return (
-    <Notice intent={intent} onClose={onClose}>
-      {children}
-    </Notice>
+    <>
+      <Button onClick={() => (!open ? setOpen(true) : setOpen(false))}>
```

Comment:
What do you think about simplifying this logic?
```suggestion
      <Button onClick={() => setOpen(!open)}>
```

Reply:
That's much cleaner, thank you!